AcsfToolsCommands.php

```diff
@@ -205,6 +210,21 @@ public function ml($cmd, $command_args = '', $command_options = '', $options = [
           // by the drush command, it can have an impact on the drupal process.
           $domain = $details['domains'][1] ?? $details['domains'][0];
 
+          // Allow specifying domain string which should be used to select exact
```

Comment:
I am not sure the comment here make sense. It looks more to explain the feature than to explain what and how the code is doing.
```
// Find a domain matching the given pattern.
```

Reply:
true, it was duplication :)

removed

Reply:
I'm avoiding the use of word pattern as we don't do regex match

Reply:
Not yet ;-)